test/conf_include_test.c

```diff
@@ -50,8 +50,10 @@ static int change_path(const char *file)
     while ((p = strpbrk(p, DIRSEP)) != NULL) {
         last = p++;
     }
-    if (last == NULL)
+    if (!TEST_ptr(last)) {
```

Comment:
Please change back to `if (last == NULL)  {`...  there's no reason whatsoever to sputter a test result if the given file doesn't have a directory component.